packages/sdk/src/stores/sorted-js-map/store.ts

```diff
@@ -39,10 +39,11 @@ export default function <Id, Data>(): SortedStore<Id, Data> {
     async set(id: Id, data: Data) {
       if (map.has(id)) {
         map.set(id, data);
+      } else {
+        const index = sortedIndex(ids, id);
+        ids.splice(index, 0, id);
+        map.set(id, data);
```

Comment:
found this bad bug, causing duplicate keys in the ids array.  this is used in api server, but have not noticed a problem. might be good to bump sdk in api once this is published.